python/S3_VPC_ENDPOINT_ENABLED/S3_VPC_ENDPOINT_ENABLED.py

```diff
@@ -95,24 +96,24 @@ def evaluate_compliance(event, configuration_item, valid_rule_parameters):
                 }
             ])
             if not endpoint_response['VpcEndpoints']:
-                annonate = 'There are no S3 endpoints present in '+ vpc_id
-                return build_evaluation(vpc_id, 'NON_COMPLIANT', event, annotation=annonate)
+                annonate = 'There are no Amazon S3 endpoints present in '+ vpc_id+'.'
+                evaluations.append(build_evaluation(vpc_id, 'NON_COMPLIANT', event, annotation=annonate))
             for endpoint in endpoint_response['VpcEndpoints']:
                 endpoint_name = endpoint['ServiceName']
                 endpoint_state = endpoint['State']
                 pattern_match = re.search(regex_pattern, endpoint_name)
                 if pattern_match and (endpoint_state == 'available'):
-                    annonate = 'S3 endpoint is present for this VPC '+ vpc_id
-                    return build_evaluation(vpc_id, 'COMPLIANT', event)
+                    annonate = 'Amazon S3 endpoint is present for this VPC '+ vpc_id+'.'
+                    evaluations.append(build_evaluation(vpc_id, 'COMPLIANT', event))
                 if pattern_match and (endpoint_state != 'available'):
-                    annonate = 'The S3 VPC endpoint is not in Available state '+vpc_id
-                    return build_evaluation(vpc_id, 'NON_COMPLIANT', event, annotation=annonate)
+                    annonate = 'The Amazon S3 VPC endpoint is not in Available state '+vpc_id+'.'
+                    evaluations.append(build_evaluation(vpc_id, 'NON_COMPLIANT', event, annotation=annonate))
                 if pattern_match is None:
-                    annonate = 'There are no S3 endpoints present in '+ vpc_id
-                    return build_evaluation(vpc_id, 'NON_COMPLIANT', event, annotation=annonate)
+                    annonate = 'There are no Amazon S3 VPC endpoints present in '+ vpc_id+'.'
+                    evaluations.append(build_evaluation(vpc_id, 'NON_COMPLIANT', event, annotation=annonate))
     except Exception as exception:
         return exception

```

Comment:
You can use the build_internal_error_response() from the template, it's used in boilerplate code as well.

Reply:
why are we using build_internal_error_response() for this ? this is a `non_compliant` evaluation and should be sent as an evaluation. Please help me understand the reasoning for this

Reply:
The suggestion was for line 115 `return exception`, you can use build_internal_error_response() or you can use build_error_response() directly as well. This is just to avoid the exception trace and return a user-friendly message.

Reply:
I've passed the exception into `build_internal_error_response()`. Let me know if this is how you wanted it